LootLockerSDK/Source/LootLockerSDK/Private/GameAPI/LootLockerPlayerRequestHandler.cpp

```diff
@@ -91,11 +91,37 @@ void ULootLockerPlayerRequestHandler::SetProfilePublic(const FResponseCallbackBP
 
 void ULootLockerPlayerRequestHandler::SetPlayerName(FString Name, const FPNameResponseBP& OnCompletedRequestBP, const FPNameResponse& OnCompletedRequest)
 {
+
+	if (ULootLockerCurrentPlatform::Get() == ELootLockerPlatform::Guest)
+	{
+
+		if (Name.Equals(ULootLockerStateData::GetPlayerIdentifier(), ESearchCase::IgnoreCase)) {
+			FLootLockerNameResponse FailResponse;
+			FailResponse.success = false;
+			FailResponse.FullTextFromServer = "Cannot set the Player name to their Identifier";
+			FailResponse.Error = FailResponse.FullTextFromServer;
+
+			OnCompletedRequestBP.ExecuteIfBound(FailResponse);
+			return;
+		}
+		if (Name.Equals("player"), ESearchCase::IgnoreCase) {
```

Comment:
This looks like it shouldn't compile, are you sure it does? Seems to me like it should be this:
```suggestion
		if (Name.Equals("player", ESearchCase::IgnoreCase)) {
```

Reply:
Ah you're right! Thank you!